README.md

```diff
@@ -7,7 +7,7 @@ Thank you for your interest in the Guardian Product & Engineering Department's c
 Inspired to work for us? [**Apply now**](https://workforus.theguardian.com/index.php/careers/product-engineering/)
 
 ## What is it?
-The coding exercise is one of the stages in our [interview process](https://www.theguardian.com/info/developer-blog/2015/jan/20/how-does-the-guardian-recruit-developers).
+The coding exercise is one of the stages in our [interview process](https://www.theguardian.com/info/2022/feb/25/how-we-have-changed-our-application-and-hiring-processes).
```

Comment:
When did the `section` (/developer-blog/) get dropped from our URLs? 🤔

Reply:
I suspect on 12/11/2020 when we renamed the tag from developer-blog to engineering-blog.  I don't know why `/engineering-blog/` didn't replace it.